cfn-resources/custom-dns-configuration-cluster-aws/cmd/resource/resource.go

```diff
@@ -45,17 +43,15 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		_, _ = logger.Warnf("Validation Error")
```

Comment:
I think there is no need for this log and maybe others

Reply:
will open a follow up